File: FutsalApi/FutsalApi.ApiService/Models/ReviewRequest.cs
using System;

namespace FutsalApi.ApiService.Models;

public class ReviewRequest
{
    public int Id { get; set; }
    public string UserId { get; set; } = string.Empty;
    public int GroundId { get; set; }
    public int Rating { get; set; }
    public string? Comment { get; set; }
    public int? ImageId { get; set; }
}

File: futsal_app/lib/view/reviews/data/model/review_request.dart
class ReviewRequest {
  final int groundId;
  final int rating;
  final String? comment;
  final int? imageId;

  ReviewRequest({
    required this.groundId,
    required this.rating,
    this.comment,
    this.imageId,
  });

  Map<String, dynamic> toJson() {
    return {
      'groundId': groundId,
      'rating': rating,
      'comment': comment,
      'imageId': imageId,
    };
  }
}
